dts/arm/nordic/nrf54l05_cpuapp.dtsi

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nordic/nrf54l05.dtsi>
+#include <nordic/nrf54l_05_10_15_cpuapp.dtsi>
```

Comment:
includes should go from less to more specific?

Reply:
The top-level include is the generic SoC, then some shared definitions specific to the cpuapp core. I think that's actually correct?